source/app/controllers/users_controller.rb

```diff
@@ -1,4 +1,38 @@
 class UsersController < ApplicationController
+  def index
+    logged_in? ? redirect_to(current_user) : redirect_to(login_path)
+  end
+
   def show
+    if owns_account? or is_admin?
```

Comment:
Rubyists tend to prefer `||` over `or`.

https://github.com/bbatsov/ruby-style-guide#no-and-or-or (worded a bit too strongly here)